### PR TITLE
Feature/207 play player attack animations

### PR DESCRIPTION
**Pull request by WildCup**
[task](https://s24109.atlassian.net/browse/SCRUM-207?atlOrigin=eyJpIjoiMGM3NGM2MWRlOTE2NDE3YTljYmY5ZWVlNzU1ZjdkM2UiLCJwIjoiaiJ9)

- separate player projectile from player body: 
player projectile and player body both have ridgitbody. When setting projectile to kinematic that let to unexpected behavior of projectile moving together with player movement. 
image

- separate projectile animations from weapon animations
since these 2 objects are now separated the animators needed to be adjusted as well
image

- setup animators and trigger animations through script:
image
image

### CODE REVIEW

**Comment by TerMyY** [comment]
I'm not sure why, but cinemachine camera on Level1 stops following the player with these changes. It works as expected on dev.

**Comment by WildCup** [comment]
player parent object now doesn't move which was desired so that it doesn't affect projectile (child) kinematic movement. With current setup camera should now follow 'player body' instead. I updated it on level 1 and 2. Thanks for testing that!

**Review by TerMyY — APPROVED** [review]

File: Assets/Scripts/Player/PlayerProjectile.cs [hunk 1]
```diff
@@ -6,16 +6,14 @@
...
     Rigidbody2D rb;
     new CircleCollider2D collider;
     [SerializeField] PlayerAttackInput inputScript;
-    bool isReady;
+    [SerializeField] Transform player;
 
-    [SerializeField] UnityEvent disabledEvent;
+    [SerializeField] UnityEvent absorbedEvent;
     [SerializeField] UnityEvent explodedEvent;
     [SerializeField] UnityEvent lunchedEvent;
```
Comment (FrozzLab) [inline Assets/Scripts/Player/PlayerProjectile.cs:18]: typo

**Review by FrozzLab — APPROVED** [review]
Very good, looks impressive in-game and the adjustability is nice. Good job, approved :>

**Pull request merged**